.gitignore

```diff
@@ -1,5 +1,4 @@
 /_site/
 /.sass-cache/
 /.project
-/.bundle/
```

Comment:
You can see the full history of the feature branch at timothyjward/osgi.enroute.site@677041944b8d50a696cff2f4de88c6cd13c0451c - it ended up in the weeds at a couple of points diverging significantly and using raw Jekyll as an attempt to pull in the source code snippets. 

Rather than pollute the history with lots of commits that wouldn't build in GitHub pages I got things building again with GitHub pages, squashed the commits, and then merged back correcting conflicts. I tried my best to avoid unnecessary deltas, but may not have caught everything.